tests/questionTypes.js

```diff
@@ -451,7 +451,7 @@ define([
 
             it("should remove appearance attribute from label", function () {
                 var from = "Trigger",
-                    to = "Text",
+                    to = "MSelect",
```

Comment:
Curious why the tests needed to be changed? I assume they no longer passed, but I'm not sure why.